docs/00b-basics/02-curriculum-structure-and-chronological-breakdown.adoc

```diff
@@ -55,32 +55,32 @@ Die Ausgestaltung der Beispiele und Übungen sind in diesem Lehrplan nicht vorge
 |===
 
 | Content
-| Duration (minutes)
-| Practice time (minutes)
```

Comment:
Kannst du "practice time" bitte wieder herstellen? Wurde von einem unserer Native Speaker damals als bester Vorschlag bewertet. 🙂  (ich hatte vorher Exercises stehen). 

Außerdem ist es dann konsistent mit den anderen Lehrplänen.

Reply:
OK, got the new wording from REQ4ARC 😅

Reply:
Because other curriculums aren't consistent, I propose to fix this later once it's clear how it should be named.